.gitignore

```diff
@@ -2,15 +2,10 @@
 .idea/
 *.dll
 *.unitypackage
-*.zip
-*.zip.meta
-*.tgz
-*.tgz.meta
 *.iml
 *.iml.meta
-SampleGameBuildLog.txt
+build.meta
 /build/
-/staging-release/
```

Comment:
Why would you want to commit build artifacts? Shouldn't they stay in gitignore?

Reply:
This is cleaning up the gitignore after I moved all the artifacts to the `build` directory